src/m365/spo/commands/site/site-recyclebinitem-restore.spec.ts

```diff
@@ -0,0 +1,117 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli';
+import Command from '../../../../Command';
+import request from '../../../../request';
+import Utils from '../../../../Utils';
+import commands from '../../commands';
+const command: Command = require('./site-recyclebinitem-restore');
+
+describe(commands.SITE_RECYCLEBINITEM_RESTORE, () => {
+
```

Comment:
Let's remove this unnecessary empty line

Reply:
done